newdle/api.py

```diff
@@ -293,6 +293,8 @@ def update_newdle(args, code):
         raise Forbidden
     for key, value in args.items():
         setattr(newdle, key, value)
+    if newdle.final_dt:
```

Comment:
`if args:` so we set the lastmod whenver something is changed? right now that's just the `final_dt` anyway...

Reply:
Fixed.